Catappult/appcoins-wallet-android
# [APPC-4359] Iap Refund Disclaimer

**What does this PR do?**

   Implement and add Refund Disclaimer in Iap Purchase, and change the layouts

**Database changed?**

   No

**How should this be manually tested?**

  Try to make a payment in Europe where have this disclaimer and change to other country outisde to europe to see the diference

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4359](https://aptoide.atlassian.net/browse/APPC-4359)

**Code Review Checklist**
